### PR TITLE
Team a

### PR DESCRIPTION
**Pull request by Adamo08**
Pull request

### CODE REVIEW

File: src/main/java/com/blockchain/model/Transaction.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import java.util.Base64;
+
+import lombok.Getter;
+import lombok.Setter;
+
+
+@Getter
+@Setter
+
+public class Transaction implements Serializable {
```
Comment (hakkm) [inline src/main/java/com/blockchain/model/Transaction.java]: here remove Serializable

File: src/main/java/com/blockchain/model/Transaction.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Base64;
+
+import lombok.Getter;
+import lombok.Setter;
+
+
+@Getter
+@Setter
```
Comment (hakkm) [inline src/main/java/com/blockchain/model/Transaction.java]: also here add a lambok @Setter for each att that needs it

File: src/main/java/com/blockchain/model/Transaction.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+public class Transaction implements Serializable {
+
+    private byte[] from;
+    private String fromFX;
```
Comment (hakkm) [inline src/main/java/com/blockchain/model/Transaction.java:21]: For example this one don't need a setter

**Review by hakkm — CHANGES_REQUESTED** [review]

**Comment by hakkm** [comment]
please do the changes, and if you have any ideas, i'm open to discuss them

**Comment by Adamo08** [comment]
> please do the changes, and if you have any ideas, i'm open to discuss them

I have reviewed and implemented the suggested changes. The issues pointed out have been addressed, and the necessary adjustments have been made. Please let me know if any further modifications are needed.

File: src/test/java/com/blockchain/model/Transaction.java
Comment (hakkm) [inline src/test/java/com/blockchain/model/Transaction.java:1]: Can you move this to src/main

File: src/main/java/com/blockchain/model/Transaction.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
+package com.blockchain.model;
+
+
+
+import sun.security.provider.DSAPublicKeyImpl;
+
+import java.io.Serializable;
```
Comment (hakkm) [inline src/main/java/com/blockchain/model/Transaction.java]: we don't need Serializable for now

File: src/main/java/com/blockchain/model/Transaction.java (hunk 1 above)
Comment (Adamo08) [reply src/main/java/com/blockchain/model/Transaction.java]: Done

File: src/test/java/com/blockchain/model/Transaction.java
Comment (Adamo08) [reply src/test/java/com/blockchain/model/Transaction.java:1]: Thanks for catching that! I've moved Transaction.java to src/main in the latest commit.

**Pull request closed without merging**